doc/examples/segmentation/plot_floodfill.py

```diff
@@ -98,7 +98,7 @@
 for i in range(8):
     m, n = np.unravel_index(i + 1, (3, 3))
     ax[m, n].imshow(output[i], cmap=plt.cm.gray)
-    ax[m, n].set_title('Tolerance {0}'.format(str(5 + 20 * i)))
+    ax[m, n].set_title(f"Tolerance {str(5 + 20 * i)}")
```

Comment:
```suggestion
    ax[m, n].set_title(f"Tolerance {5 + 20 * i}")
```
can also remove the `str()` call here